rebar.config

```diff
@@ -78,6 +78,7 @@
 {dialyzer, [ {warnings, [ unknown
                         ]}
            , {plt_apps, all_deps}
+           , {plt_extra_apps, [mnesia]}
```

Comment:
Due to the Mnesia setup (the application needs to be stopped when creating the schema), mnesia is not part of the erlang_ls `apps`. Therefore, Dialyzer would not include it as a dependency automatically, resulting in "unknown functions" warnings.